insights-notification-schemas-java/src/test/java/com/redhat/cloud/notifications/ingress/TestSerialization.java

```diff
@@ -109,7 +145,7 @@ void encodeAndDecodeV1_0_0() {
         action.put("events", List.of());
 
         // This version does not have a "version" field
-        // Assertions.assertThrows(AvroRuntimeException.class, () -> action.put("version", "v1.0.0"));
+        Assertions.assertThrows(AvroRuntimeException.class, () -> action.put("version", "v1.0.0"));
```

Comment:
all the static Assertions.* could be changed to static imports. Also not that important but improves readability a bit